src/components/dashboard/defects/DefectTable11.jsx

```
@@ -0,0 +1,376 @@
+  ), [windowWidth]);
+
+  // Actions column
+  const actions = useMemo(() => ({
+    label: '',
+    width: '60px',
+    minWidth: '60px',
+    content: (defect) => (
+      <div className="flex justify-center">
+        {permissions.actionPermissions.delete && (
+          <button
+            onClick={e => { e.stopPropagation(); onDelete && onDelete(defect); }}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)